css/main.css

```diff
@@ -125,8 +125,13 @@ body {
 
 .social-media-logo {
   vertical-align: middle;
-  width: 15px;
-  height: 15px;
+  width: 35px;
+  height: 35px;
+  transition: all 0.5s ease-in-out;
+}
+
+.social-media-logo:hover {
+  transform: scale(1.3);
```

Comment:
Can we change it to `1.25` ??
The reference was only for context, how the end outcome will look like.
Using values in multiples or in ratio scales is a good practice, like `1.25`, `1.5`, `1.75`.

Reply:
Sure thing I'll change that right away

Reply:
Alright just fixed issue with the checks and there aren't anymore failures. Also, changed 1.3 --> 1.25 for the icon scaling. Let me know if there's anything else you want me to change